.storybook/assets/base.css

```diff
@@ -11,6 +11,15 @@
  * governing permissions and limitations under the License.
  */
 
+ :root {
```

Comment:
Can you attach these to the .spectrum class? I'm working on making sure any code in storybook that should look "spectrum" is wrapped in the appropriate spectrum classes.

Reply:
This is done in the latest update.